### DISCUSSION TITLE
Chat

### DISCUSSION THREAD

**Discussion by PranaMohanty13** [flyerhq/flutter_chat_ui#460] [Q&A] [answered]
i am getting the following error: "The method 'Chat' isn't defined for the type '_ChatPageState'.
Try correcting the name to the name of an existing method, or defining a method named 'Chat'."... at the Widget build(BuildContext context) => Scaffold

        body: Chat(
          messages: _messages,
          onAttachmentPressed: _handleAttachmentPressed,
          onMessageTap: _handleMessageTap,
          onPreviewDataFetched: _handlePreviewDataFetched,
          onSendPressed: _handleSendPressed,
          showUserAvatars: true,
          showUserNames: true,
          user: _user,
        ),
      );..... how should I fix this?

**Comment by demchenkoalex** [comment, marked as answer]
Can you post more code? the one you have sent looks ok. Try example, does it work? Try the same config as in there if it works
